drivers/shared/executor/executor_linux.go

```diff
@@ -319,10 +319,10 @@ func (l *LibcontainerExecutor) Shutdown(signal string, grace time.Duration) erro
 		case <-l.userProcExited:
 			return nil
 		case <-time.After(grace):
-			return l.container.Signal(os.Kill, false)
+			return l.container.Signal(os.Kill, true)
 		}
 	} else {
-		return l.container.Signal(os.Kill, false)
+		return l.container.Signal(os.Kill, true)
```

Comment:
Add a comment because the boolean literals are totally opaque to readers unless they have the docs handy.